src/ucs/type/status.h

```diff
@@ -80,6 +80,9 @@ typedef enum {
     UCS_ERR_LAST                   = -100
 } UCS_S_PACKED ucs_status_t ;
 
+
+#define UCS_ERR_PENDING (UCS_ERR_LAST - 1)
```

Comment:
it should be defined in UCP and named UCP_STATUS_PENDING_SWITCH (or similar), since the pending queue manipulation is specific for UCP module only

Reply:
renamed & moved to proto file